app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :post, foreign_key: 'post_id'
+  belongs_to :author, class_name: 'User'
+
+  validates :text, presence: true
+
+  after_create :increment_post_comments_counter
+
+  def increment_post_comments_counter
```

Comment:
- [ ] @hetrox8, great effort working on your project but i would kindly suggest you make use of the `after_save` Rails Active Callbacks to call the `increment_post_comments_counter` instead of the `after_create` you used. 

e.g
```ruby
  after_save :increment_post_comments_counter

  def increment_post_comments_counter
    post.increment!(:comments_counter)
  end
```